tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>Reminder to replace references to <code>modulate_det_map</code> in the tutorial too </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>